python/pylibcugraph/pylibcugraph/experimental/__init__.py

```diff
@@ -71,3 +71,6 @@
 
 from pylibcugraph.katz_centrality import EXPERIMENTAL__katz_centrality
 katz_centrality = experimental_warning_wrapper(EXPERIMENTAL__katz_centrality)
+
+from pylibcugraph.eigenvector_centrality import EXPERIMENTAL__eigenvector_centrality
+eigenvector_centrality = experimental_warning_wrapper(EXPERIMENTAL__eigenvector_centrality)
```

Comment:
Reminder: don't forget to also drop the `EXPERIMENTAL__` prefix after promoting.